labs/lab001/lab001.md

```diff
@@ -22,8 +34,8 @@ This lab should be performed on **YOUR ASSIGNED INSTANCE** only, as the *kubevir
 
 **NOTE**: Please be respectful and only connect to your assigned instance. All instances on this lab use the **same** public key, so you could accidentally connect to the wrong system. If you have any issues, please inform your instructor/s.
 
-```console
-$ ssh -i kubevirt-tutorial kubevirt@kubevirtlab-<number>.try.kubevirt.me
+```
+$ ssh -i kubevirt-tutorial kubevirt@kubevirtlab-<number>.gce.sexylinux.net
```

Comment:
This reverts a previous change that AFAIK shouldn't be reverted... the `kubevirt.me` domain was created for this purpose

Reply:
Agree, done!